lib/phoenix/live_dashboard/system_info.ex

```diff
@@ -44,10 +44,19 @@ defmodule Phoenix.LiveDashboard.SystemInfo do
 
   ## Fetchers
 
-  def fetch_processes(node, search, sort_by, sort_dir, limit, prev_reductions \\ nil) do
+  def fetch_processes(
+        node,
+        process_filter,
```

Comment:
We can call it just filter; we are in the `fetch_processes` function :)

Reply:
Thanks!  Fixed by [a74954b](https://github.com/phoenixframework/phoenix_live_dashboard/pull/411/commits/a74954b6cc230d9ccf366665de5138d887134208)